includes/omise-wc-setting.php

```diff
@@ -70,36 +71,36 @@
 		'label'       => Omise_Card_Image::get_amex_image(),
 		'css'         => Omise_Card_Image::get_css(),
 		'default'     => Omise_Card_Image::get_amex_default_display(),
-		'description' => __( 'This only controls the icons displayed on the checkout page.<br />It is not related to card processing on Omise payment gateway.', $text_domain )
+		'description' => Omise_Util::translate( 'This only controls the icons displayed on the checkout page.<br />It is not related to card processing on Omise payment gateway.' )
 	),
 	'title' => array(
-		'title'       => __( 'Title', $text_domain ),
+		'title'       => Omise_Util::translate( 'Title', 'Setting title' ),
 		'type'        => 'text',
-		'description' => __( 'This controls the title which the user sees during checkout.', $text_domain ),
-		'default'     => __( 'Omise Payment Gateway', $text_domain )
+		'description' => Omise_Util::translate( 'This controls the title which the user sees during checkout.' ),
+		'default'     => Omise_Util::translate( 'Omise Payment Gateway', 'Default title' )
 	),
 	'payment_action' => array(
-		'title'       => __( 'Payment Action', $text_domain ),
+		'title'       => Omise_Util::translate( 'Payment action' ),
 		'type'        => 'select',
-		'description' => __( 'Manual Capture or Capture Automatically', $text_domain ),
+		'description' => Omise_Util::translate( 'Manual Capture or Capture Automatically' ),
 		'default'     => 'auto_capture',
 		'class'       => 'wc-enhanced-select',
 		'options'     => array(
-			'auto_capture'   => __( 'Auto Capture', $text_domain ),
-			'manual_capture' => __( 'Manual Capture', $text_domain )
+			'auto_capture'   => Omise_Util::translate( 'Auto Capture', 'Setting auto capture' ),
+			'manual_capture' => Omise_Util::translate( 'Manual Capture', 'Setting manual capture' )
 		),
 		'desc_tip'    => true
 	),
 	'omise_3ds' => array(
-		'title'       => __( '3DSecure Support', $text_domain ),
+		'title'       => Omise_Util::translate( '3-D Secure support' ),
 		'type'        => 'checkbox',
-		'label'       => __( 'Enables 3DSecure on this account (does not support for Japan account)', $text_domain ),
+		'label'       => Omise_Util::translate( 'Enable or disable 3-D Secure for the account. (Japan-based accounts are not eligible for the service.)' ),
 		'default'     => 'no'
 	),
 	'description' => array(
-		'title'       => __( 'Description', $text_domain ),
+		'title'       => Omise_Util::translate( 'Description', 'Setting description' ),
```

Comment:
just checked, usually it would shows there 👇  
<img width="835" alt="screen shot 2559-08-19 at 2 08 04 am" src="https://cloud.githubusercontent.com/assets/2154669/17787105/ef3a3fec-65b1-11e6-9b64-10ffbfa19a35.png">

(ref: plugin woocommerce/includes/abstracts/abstract-wc-payment-gateway.php line: 338)

btw, we overrode that method by implementing `payment_fields` method by ourself (here https://github.com/omise/omise-woocommerce/blob/master/omise-wc-gateway.php#L69)

so, no one used it right now... we can remove that setting from the setting page (could be another PR)